source/sonata_tech.rst

```diff
@@ -576,3 +576,23 @@ Required fields for ``Spykfunc``:
 .. _specification: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
 .. _enumeration: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#nodes---enum-datatypes
 .. _Tsodyks Markram Model: https://www.pnas.org/content/94/2/719
+
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ===================================== ======================= ========== =================== ============= ==========================================
+  Group                                 Field                   Type       Shape               Requirement   Description
+  ===================================== ======================= ========== =================== ============= ==========================================
+  /electrodes/contacts/{electrodename}  layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/contacts/{electrodename}  position                float32    3                   Mandatory     Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/contacts/{electrodename}  region                  utf8       1                   Optional      Region in which {electrodename} is located
+  /electrodes/contacts/{electrodename}  type                    utf8       1                   Optional      Either EEG or LFP
+  /electrodes/{population_name}         data                    float64    Total_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
```

Comment:
How are multiple electrodes tied together, if they reside in the same population?  IE if we have
```
/electrodes/contacts/electrode0
/electrodes/contacts/electrode1
```
That both refer to `nodeA`, how does one know how they correspond to: `/electrodes/{population_name}` and `/{population_name}`?

Reply:
I've added an index field in the latest commit